src/System.Collections/src/Resources/Strings.resx

```diff
@@ -202,7 +193,7 @@
     <value>Target array type is not compatible with the type of items in the collection.</value>
   </data>
   <data name="ArgumentOutOfRange_BiggerThanCollection" xml:space="preserve">
-    <value>Larger than collection size.</value>
+    <value>Must be less than or equal to the size of the collection.</value>
```

Comment:
For the second I'd think ArgumentOutOfRange_Count would be better, with the error mentioning "count" rather than "index".  Argument_InvalidOffLen doesn't work in part because it refers to names that don't exist in this function.